### ISSUE TITLE
CUSA26376 - NEO : The World Ends with You

### ISSUE THREAD

**Issue by StevenMiller123** [shadps4-emu/shadps4-game-compatibility#1467]
### Game Name

NEO : The World Ends with You

### Game code

CUSA26376

### Game version

v01.00

### Used emulator's version (only released versions are acceptable)

v0.5.0

### Current status

Nothing

### Operating System

Windows

### Error

```
[Debug] <Critical> memory.cpp:operator():387: Assertion Failed!
Existing mapping does not contain requested unmap range
```

### Description

Crashes instantly

### Log File

[shad_log.txt](https://github.com/user-attachments/files/18249053/shad_log.txt)

**Comment by uncle-ben-devel** [comment]
Tested the EU variant (CUSA-26377) with 0.6.0 and 11.00 sysmodules.
Do we do separate issues since the CUSA code is different?
Observations:
- Main menu works
- Sound in main menu works
- Crashes on starting a new game (it appears it tries to load lib_burst_generated, which does not exist)

![Image](https://github.com/user-attachments/assets/5a0d1d51-f324-4f26-a5e9-00a8e2030e8c)

![Image](https://github.com/user-attachments/assets/7d10dc07-57ec-4d06-b1cd-f84d9b2b6759)

[shad_log-CUSA-26377.txt](https://github.com/user-attachments/files/18661281/shad_log-CUSA-26377.txt)

**Comment by StevenMiller123** [comment]
@uncle-ben-devel Yes, your report should go to an issue for your CUSA. If no one has done a report on that CUSA of the game, please create a new issue report.

**Comment by StevenMiller123** [comment]
Reaches menus on v0.6.0, trying to go ingame causes a `std::out_of_range` exception during a sceKernelLoadStartModule call.

![Image](https://github.com/user-attachments/assets/c2b6fe61-747b-4ab4-ba9b-8a37666831b1)

![Image](https://github.com/user-attachments/assets/164fe1f1-6087-426c-91a7-51ec208f4e27)

[shad_log.txt](https://github.com/user-attachments/files/18726699/shad_log.txt)
